tests/helpers.py

```diff
@@ -72,10 +68,8 @@ def get_usb_clk_phy(
             "XS1_USB_TERMSEL",
             clk,
             verbose=verbose,
-            test_ctrl=test_ctrl,
             do_timeout=do_timeout,
             complete_fn=complete_fn,
-            expect_loopback=expect_loopback,
             dut_exit_time=dut_exit_time,
         )
 
```

Comment:
In line 223, 
`if bool(os.getenv("enabletracing")):`  will also be True.

Reply:
thanks!